src/core/public/chrome/ui/header/header_logo.tsx

```diff
@@ -85,12 +87,15 @@ function onClick(
 
 interface Props {
   href: string;
-  navLinks: NavLink[];
-  forceNavigation: boolean;
+  navLinks$: Observable<ChromeNavLink[]>;
+  forceNavigation$: Observable<boolean>;
   navigateToApp: (appId: string) => void;
 }
 
-export function HeaderLogo({ href, forceNavigation, navLinks, navigateToApp }: Props) {
+export function HeaderLogo({ href, navigateToApp, ...observables }: Props) {
+  const forceNavigation = useObservable(observables.forceNavigation$, false);
+  const navLinks = useObservable(observables.navLinks$, []);
```

Comment:
NIT: not sure to see a real upside in using a rest parameter here, but I'm fine either way